src/App.ts

```diff
@@ -558,6 +602,13 @@ export default class App {
    */
   public async processEvent(event: ReceiverEvent): Promise<void> {
     const { body, ack } = event;
+
+    if (this.developerMode) {
+      // log the body of the event
+      // this may contain sensitive info like tokens
+      this.logger.debug(JSON.stringify(body));
```

Comment:
I stringify so we can see all of the nested objects

Reply:
Just a suggestion. This can be a global middleware as I do in my personal template project: https://github.com/seratch/bolt-starter/blob/55bf3bac47d7f94d9b868ddd0314782aef6d7245/index.js#L28-L47

You can "use" a middleware for request logging in the constructor. We may want to expose the middleware for developers who just want to use it without developer mode.

Reply:
That is a good suggestion. The nice thing about its current placement is that it is one of the first things that happens. In case middleware was broken or some other error happens, the event could potentially not get logged if it was in middleware. 

Lets keep discussing as this is something we can change in a minor release.